lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -129,55 +253,121 @@
 
         if self.params.get('state') == 'present':
-            # Gather information about all vSwitches and Physical NICs
+            # Gather information about all vSwitches
             network_manager = self.host_system.configManager.networkSystem
             available_pnic = [pnic.device for pnic in network_manager.networkInfo.pnic]
             self.available_vswitches = dict()
             for available_vswitch in network_manager.networkInfo.vswitch:
+                vswitch_security = dict()
+                vswitch_security['allowPromiscuous'] = available_vswitch.spec.policy.security.allowPromiscuous
+                vswitch_security['macChanges'] = available_vswitch.spec.policy.security.macChanges
+                vswitch_security['forgedTransmits'] = available_vswitch.spec.policy.security.forgedTransmits
+                vswitch_teaming = dict()
+                vswitch_teaming['policy'] = available_vswitch.spec.policy.nicTeaming.policy
+                vswitch_teaming['notifySwitches'] = available_vswitch.spec.policy.nicTeaming.notifySwitches
+                vswitch_teaming['rollingOrder'] = available_vswitch.spec.policy.nicTeaming.rollingOrder
+                vswitch_teaming['reversePolicy'] = available_vswitch.spec.policy.nicTeaming.reversePolicy
+                vswitch_teaming['failover_order_active_adapters'] = \
+                    available_vswitch.spec.policy.nicTeaming.nicOrder.activeNic
+                vswitch_teaming['failover_order_standby_adapters'] = \
+                    available_vswitch.spec.policy.nicTeaming.nicOrder.standbyNic
+                vswitch_failure = dict()
+                vswitch_failure['checkSpeed'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.checkSpeed
+                vswitch_failure['speed'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.speed
+                vswitch_failure['checkDuplex'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.checkDuplex
+                vswitch_failure['fullDuplex'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.fullDuplex
+                vswitch_failure['checkErrorPercent'] = \
+                    available_vswitch.spec.policy.nicTeaming.failureCriteria.checkErrorPercent
+                vswitch_failure['percentage'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.percentage
+                vswitch_failure['checkBeacon'] = available_vswitch.spec.policy.nicTeaming.failureCriteria.checkBeacon
+                vswitch_ts = dict()
+                vswitch_ts['enabled'] = available_vswitch.spec.policy.shapingPolicy.enabled
+                vswitch_ts['averageBandwidth'] = available_vswitch.spec.policy.shapingPolicy.averageBandwidth
+                vswitch_ts['peakBandwidth'] = available_vswitch.spec.policy.shapingPolicy.peakBandwidth
+                vswitch_ts['burstSize'] = available_vswitch.spec.policy.shapingPolicy.burstSize
+                vswitch_offload = dict()
```

Comment:
please use dict literal
```suggestion
                vswitch_offload = {}
```

Reply:
Is there a specific reason for this recommendation?